ISLE/isleapp.cpp

```diff
@@ -113,8 +118,8 @@ IsleApp::IsleApp()
 	m_drawCursor = FALSE;
 	m_use3dSound = TRUE;
 	m_useMusic = TRUE;
-	m_useJoystick = TRUE;
-	m_joystickIndex = 0;
+	m_useGamepad = TRUE;
```

Comment:
We should not rename the variables, function names etc from Joystick to Gamepad. Let's keep what we had. This will facilitate easier merging and avoid confusion. Right now half of the code uses gamepad terminology, the other joystick. Let's settle on the original joystick wording.

Reply:
Got it, I'll rename those back to joystick